### PR TITLE
feature: Add dynamic slots to ViTable

### PR DESCRIPTION
**Pull request by beeblebrox3**
## Description

This PR add dynamic slots to ViTable and simple default render for cells. Now is possible to render a simple table without the need to code `<td>` and customize individual cells if necessary.

## Before & After Screenshots

**BEFORE**:
[insert screenshot here]
image

**AFTER**:
personalizacao-slot
tabela-basica

### CODE REVIEW

File: src/components/Table.vue [hunk 1]
```diff
@@ -415,10 +421,76 @@ export default {
...
+  export default {
+    data() {
+      return {
+        columns: [
+          { id: 'id', label: 'ID' },
+          { id: 'name', label: 'Name' },
+          { id: 'email', label: 'E-mail' },
+        ],
+        data: [
+          { id: 1, name: 'Amanda', email: 'amanda@vitta.com.br' },
```
Comment (jeancarlos) [inline src/components/Table.vue]: Recomendo usar um domínio fake, por causa de crawlers de email pra spam, mas não impede de aprovar o PR e mesclar na master.

Comment (beeblebrox3) [reply src/components/Table.vue]: you are absolutely right! I'll fix it

Comment (beeblebrox3) [reply src/components/Table.vue]: Done!

**Comment by taciomedeiros** [comment]
@beeblebrox3 @luizzzdev @jeancarlos  this is a breaking change without backward compatibility. How we will handle this type of change?

**Comment by beeblebrox3** [comment]
@taciomedeiros  I don't think it is. We just add new slots. A new minor should be sufficient.

**Comment by taciomedeiros** [comment]
Great!

**Comment by jeancarlos** [comment]
This is not breaking change as @beeblebrox3 said. But when it happens we will go up a minor. But everything should be made to the lowest version possible. If a feature is made for 0.1.2 but it works on 0.0.18. The feature should be made to 0.0.19 and 0.1.3.

**Pull request merged**